flink-runtime/src/main/java/org/apache/flink/runtime/state/StateBackend.java

```diff
@@ -186,7 +187,8 @@
 			kvStateRegistry,
 			ttlTimeProvider,
 			new UnregisteredMetricsGroup(),
-			Collections.emptyList());
+			Collections.emptyList(),
+			new CloseableRegistry());
```

Comment:
This looks somehow wrong to do. I found that this method is (maybe transitively) only really reached by tests. I suggest to remove the method and call the full method to which it delegates in the tests instead.

Reply:
Ok, let me fix this in latter commits to save some efforts for code rebase. Thanks.